JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -209,9 +211,13 @@ fun ErrorSnackbar(
                     onClick = {
                         onErrorAction()
                         onDismiss()
-                    }
+                    },
+                    contentColor = contentColor()
                 ) {
-                    Text("RETRY")
+                    Text(
+                        text = "RETRY",
+                        color = Red300
```

Comment:
Code smell using a hardcoded color?

Reply:
Yeah, design calls for non-theme values (b/155218328). Snackbars are weird. Will fix.